### PR TITLE
Add city name when available

### LINKED ISSUE #877: City Name is not displayed , only country name is displayed for a Post( Feed->Post)
Summary : City Name is not displayed , only country name is displayed for a Post( Feed->Post)

Steps:
Go to Url,  https://staging.fightpandemics.work/feed

Actual Result: Only country name is displayed for a Post
post

Expected Result:
As per zeplin,City name is displayed along with country name
https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5ed034ab3d8c25b62217f26f

### PR DESCRIPTION
**Pull request by ac5144**
### What does this pull request aim to achieve? 💯
Closes #877 

Added a line to display the city name of Posts in the Feed when available.
_Please be concise and link any related issue(s):_
I added a line to check and display the city of a 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/components/Feed/Post.js [hunk 1]
```diff
@@ -15,10 +15,7 @@ import { LOGIN } from "templates/RouteWithSubRoutes";
 import PostCard from "./PostCard";
 import PostSocial from "./PostSocial";
 import SubMenuButton from "components/Button/SubMenuButton";
-import WizardFormNav, {
-  StyledButtonWizard,
-} from "components/StepWizard/WizardFormNav";
-import { StyledLoadMoreButton } from "./StyledCommentButton";
+import WizardFormNav, { StyledButtonWizard } from "components/StepWizard/WizardFormNav";
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:18]: Please check your prettier configurations/ ESLint configurations

File: client/src/components/Feed/Post.js [hunk 2]
```diff
@@ -35,8 +32,6 @@ import {
 import SvgIcon from "../Icon/SvgIcon";
 import statusIndicator from "assets/icons/status-indicator.svg";
 
-const INDIVIDUAL_AUTHOR_TYPE = "Individual";
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:38]: Why are we deleting this?

File: client/src/components/Feed/Post.js [hunk 3]
```diff
@@ -45,7 +40,6 @@ const Post = ({
   fullPostLength,
   handlePostDelete,
   handlePostLike,
-  includeProfileLink,
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:48]: same comment as previous one

File: client/src/components/Feed/Post.js [hunk 4]
```diff
@@ -79,9 +73,6 @@ const Post = ({
       post.author.name.match(/\b\w/g).join("").toUpperCase()) ||
     "";
 
-  const authorProfileLink = `/${
-    post.author.type === INDIVIDUAL_AUTHOR_TYPE ? "profile" : "organization"
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:83]: ? why is this being deleted?

File: client/src/components/Feed/Post.js [hunk 5]
```diff
@@ -192,32 +182,6 @@ const Post = ({
...
-          SET_COMMENTS,
-          "comments",
-          filterComments,
-          "numComments",
-          filterComments.length,
-        );
-      }
-    }
-  };
-
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:220]: Any reason of this being deleted?

File: client/src/components/Feed/Post.js [hunk 6]
```diff
@@ -274,16 +238,13 @@ const Post = ({
...
+          {post?.author?.location?.country ? post.author.location.country : ""}
         </span>
       }
     />
   );
 
-  const renderHeaderWithLink = (
-    <Link to={authorProfileLink}>{renderHeader}</Link>
-  );
-
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:286]: ? same as previous comment?

**Comment by ac5144** [comment]
That's weird, I only added a line and didn't delete anything. Sorry, let me check.

**Comment by idjevm** [comment]
Yeah probably because you were behind in master ? or something like that ?

**Comment by idjevm** [comment]
> That's weird, I only added a line and didn't delete anything. Sorry, let me check.

This might help how to check changes before opening the PR -> https://github.com/FightPandemics/FightPandemics/wiki/How-to-create-a-Pull-Request

**Comment by ac5144** [comment]
> > That's weird, I only added a line and didn't delete anything. Sorry, let me check.
> 
> This might help how to check changes before opening the PR -> https://github.com/FightPandemics/FightPandemics/wiki/How-to-create-a-Pull-Request

Should I just create a new PR or keep updating this one?

**Comment by idjevm** [comment]
this one is fine :) No need to open a new PR

**Comment by ac5144** [comment]
The previous commit was most likely based on an outdated master. I've rebased and it looks like only two lines should be changed now.

File: client/src/components/Feed/Post.js [hunk 7]
```diff
@@ -274,7 +274,8 @@ const Post = ({
       extra={
         <span>
           <SvgIcon src={statusIndicator} className="status-icon" />
-          {post?.author?.location ? post.author.location.country : ""}
+		  {post?.author?.location?.city ? `${post.author.location.city}, ` : ""}
+          {post?.author?.location?.country ? post.author.location.country : ""}
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:278]: Really weird formatting, Check your prettier settings, and make sure you have ESLint configuration set to true on the .env file for the client. Please reformat.

File: client/src/components/Feed/Post.js [hunk 8]
```diff
@@ -482,7 +487,9 @@ const Post = ({
                 onClick={onClick}
                 loadMorePost={loadMorePost}
               />
-            ): (<Card.Body className="view-more-wrapper"/>))}
+            ) : (
+              <Card.Body className="view-more-wrapper" />
+            ))}
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:492]: Reformatting applied by me :) Should be good now

File: client/src/components/Feed/Post.js [hunk 9]
```diff
@@ -418,7 +421,9 @@ const Post = ({
                 onClick={onClick}
                 loadMorePost={loadMorePost}
               />
-            ): (<Card.Body className="view-more-wrapper"/>)}
+            ) : (
+              <Card.Body className="view-more-wrapper" />
+            )}
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:426]: Reformatting applied by me :) Should be good now

File: client/src/components/Feed/Post.js [hunk 10]
```diff
@@ -274,7 +276,8 @@ const Post = ({
       extra={
         <span>
           <SvgIcon src={statusIndicator} className="status-icon" />
-          {post?.author?.location ? post.author.location.country : ""}
+          {post?.author?.location?.city ? `${post.author.location.city}, ` : ""}
+          {post?.author?.location?.country ? post.author.location.country : ""}
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:280]: Reformatting applied by me :) Should be good now

File: client/src/components/Feed/Post.js [hunk 11]
```diff
@@ -205,7 +205,9 @@ const Post = ({
         console.log({ error });
       }
       if (response && response.data) {
-        let filterComments = comments.filter((comment) => comment._id !== commentId)
+        let filterComments = comments.filter(
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:208]: Reformatting applied by me :) Should be good now

File: client/src/components/Feed/Post.js [hunk 12]
```diff
@@ -1,6 +1,6 @@
 // Core
 import React, { useEffect, useState, useRef } from "react";
-import { Button, Modal as WebModal } from "antd";
+import { Modal as WebModal } from "antd";
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:3]: removed unused import

**Pull request merged**